jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/PostControllerTest.java

```diff
@@ -223,13 +224,13 @@ public void testSubmitAnswerValidationFail() throws NotFoundException {
         BeanPropertyBindingResult resultWithErrors = mock(BeanPropertyBindingResult.class);
         when(resultWithErrors.hasErrors()).thenReturn(true);
         //invoke the object under test
-        ModelAndView mav = controller.create(null, TOPIC_ID, getDto(), resultWithErrors);
+        ModelAndView mav = controller.create(null, TOPIC_ID, getDto(), resultWithErrors, new RedirectAttributesModelMap());
 
         //check expectations
         verify(topicModificationService, never()).replyToTopic(anyLong(), anyString(), eq(BRANCH_ID));
 
         //check result
-        assertViewName(mav, "topic/postList");
+        assertEquals(mav.getViewName(), "redirect:/topics/error/" + TOPIC_ID + "?page=null");
```

Comment:
@standpoint , this test is called testSubmitAnswerValidationFail(), thus it checks redirect after validation fail as well as it did before. The only difference is that previously it was redirect to the view and now it is redirect to another action. Do you think I have to write any extra tests? If so, please, let me know what you want me to check.